bouquet/assess.py

```
@@ -82,4 +88,9 @@ def relax_structure(atoms: Atoms, energyCalc: Calculator, calc: Calculator, step
     except ValueError:  # LBFGS failed to converge, probably high energy
         pass
 
-    return energyCalc.get_potential_energy(atoms), atoms
+    try:
+        energy = energyCalc.get_potential_energy(atoms)
+    except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)